projects/mpmath.org/package.yml

```diff
@@ -0,0 +1,24 @@
+distributable:
+  url: https://mpmath.org/files/mpmath-{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  url: https://mpmath.org
+  match: /mpmath-\d+\.\d+\.\d+\.tar\.gz/
+  strip:
+    - /^mpmath-/
+    - /\.tar\.gz$/
+
+dependencies:
+  python.org: '>=3.11'
```

Comment:
Python breaks between minor versions. In no case should we depend on it like this. Always depend on python with `~` eg. `python.org: ~3.11`